docs/_instructions/includeresource.md

```diff
@@ -70,6 +70,20 @@ Wrapping often requires access to a JAR from the repository. It is therefore com
 
     -includeresource    @${repo;biz.aQute.bndlib}!/about.html
 
+### Unrolling with extras
+
+`flatten:=BOOLEAN` - puts all files in the file-tree into one folder
+
+    -includeresource new.package=@jar/file.jar!/META-INF/cxf/**;flatten:=true
+
+`rebase:=PATH` - rebases the path to the file to the given value, regex groups allowed on folders
```

Comment:
They are not really regex, they are instruction selectors. https://bnd.bndtools.org/chapters/820-instructions.html#selector